changelog.d/20231024_123046_boris_keypoints.md

```diff
@@ -0,0 +1,5 @@
+### Added
+
+- Keypoint detector supporting skeletons with following classes: body, head, foot, hands. 
```

Comment:
This changelog entry seems to bury the lead, which is that serverless functions can now return skeleton annotations.
That seems like something worth highlighting.